backend/lib/endpoints/organizations.js

```diff
@@ -0,0 +1,95 @@
+const httpErrors = require("http-errors");
+
+const {
+  createOrganizationSchema,
+  deleteOrganizationSchema,
+  getOrganizationSchema,
+  getOrganizationsSchema,
+  updateOrganizationSchema,
+} = require("./schema/organizations");
+
+/*
+ * /api/organizations
+ */
+async function routes(app) {
+  const Organization = app.mongo.model("Organization");
+
+  app.delete(
+    "/:organizationId",
+    {
+      preValidation: [app.authenticate],
+      schema: deleteOrganizationSchema,
+    },
+    async (req) => {
+      // TODO: make sure user can only delete organizations they own
+      return Organization.findByIdAndRemove(req.params.organizationId);
+    },
+  );
+
+  app.get(
+    "/",
+    {
+      preValidation: [app.authenticate],
+      schema: getOrganizationsSchema,
+    },
+    async (req, reply) => {
+      const { ownerId } = req.params;
+      const filter = ownerId ? { ownerId } : {};
+      const sortedOrganizations = await Organization.find(filter).sort({
+        name: 1,
+      });
+      return reply.send(sortedOrganizations);
```

Comment:
Not super familiar with Fastify but is there a reason why we're sending instead of returning `sortedOrganizations` alone?

Reply:
Thanks for catching this, I actually copied this from `posts.js`. I agree with the suggestion that we just return the object literal or error object directly rather than use `reply.send`. Will fix this.

Reply:
Fixed in e17fb45